## ⚠️ Pre-existing failure on `main` is blocking CI green here

CI on this PR is failing on a **`@agor/daemon`** test (NOT `@agor/core`):

```
FAIL src/services/users.git-env.test.ts > UsersService.getGitEnvironment — permission checks > user JWT cannot fetch another user env
AssertionError: expected [Function] to throw error matching /Forbidden/ but got 'Cannot access another user's git env…'
```

This same test fails on `main` ([CI run on 1f3dc43b](https://github.com/preset-io/agor/actions/runs/25402941939)). #1088 ("fix(git): per-user impersonated clone & cred plumbing in strict mode") appears to have changed the rejection error message from one matching `/Forbidden/` to `\"Cannot access another user's git environment\"`. The security behavior is still correct (cross-user access still rejected); only the assertion's regex needs to match the new message.

One-line fix on `apps/agor-daemon/src/services/users.git-env.test.ts:75` (loosen the regex to match the new error string), but it's outside the scope of \"re-enable @agor/core tests\" and existed on `main` before this branch. **How would you like to handle it?**

- (a) I add the one-line fix to this PR
- (b) Land a separate tiny PR for the daemon test first, then this rebases green

`@agor/core` itself is clean — 62/62 files, 2083/2083 tests, 3× locally stable. The local-green / CI-red gap is solely the daemon assertion.

--- a/apps/agor-daemon/src/services/users.git-env.test.ts
+++ b/apps/agor-daemon/src/services/users.git-env.test.ts
@@ -72,7 +72,9 @@ describe('UsersService.getGitEnvironment — permission checks', () => {
       },
     };
 
-    await expect(service.getGitEnvironment({ userId: userB }, params)).rejects.toThrow(/Forbidden/);
+    await expect(service.getGitEnvironment({ userId: userB }, params)).rejects.toThrow(
+      /Cannot access another user's git environment/
+    );
   });
 
   dbTest('unauthenticated caller is rejected', async ({ db }) => {

